src/Makefile

```diff
@@ -202,7 +202,7 @@ endif
 
 # redis-server
 $(REDIS_SERVER_NAME): $(REDIS_SERVER_OBJ)
-	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a ../deps/lua/src/liblua.a $(FINAL_LIBS)
+	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a ../deps/lua/src/liblua.a ../deps/memkind/.libs/libmemkind.a ../deps/memkind/jemalloc/obj/lib/libjemalloc.a ../deps/jemalloc_JE/lib/libjemalloc.a $(FINAL_LIBS) -lnuma
```

Comment:
Consider in linking static library usage of variables:
https://github.com/pmem/redis/blob/8856349a6319d148bff8ea2528a35328b85101e8/src/Makefile#L26-L27